tests/project/test_mbed_project.py

```diff
@@ -26,9 +39,8 @@ def test_skips_mbed_os_when_create_only_is_true(self, mock_program):
         mock_program.from_new.return_value.resolve_libraries.assert_not_called()
 
 
-@mock.patch("mbed_tools.project.project.MbedProgram", autospec=True)
-class TestCloneProject(TestCase):
-    def test_clones_from_remote(self, mock_program):
+class TestCloneProject:
+    def test_clones_from_remote(self, mock_git):
```

Comment:
`test_clones_from_remote()` used to use `mock_program`. In commit "mbed_project: Remove unittest.TestCase from tests", do we want to move it to use `mock_git` as is done here, or should we keep it using `mock_program`? This probably should use `mock_program` if we want this commit to pass tests.

Reply:
Fixed.